reference-guides/collections-granules.qmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Collections & Granules"
+format:
+  html:
+    toc: true
+    code-fold: false
+execute:
+  enabled: false
+---
+
+NASA Earth science data can feel overwhelming when you're new to it... there are thousands of datasets, millions of files, and many tools for accessing them.  A first step in combatting the overhwhelm is better understanding how NASA organizes, and talks about, their data. 
+
+---
+
+# What Are Collections and Granules?  
+
+## 📘 Collections  
+
+A **collection** is NASA’s version of a “dataset family.”  
+Every collection groups together many related files that share common features, such as:
+
+- The same satellite or instrument  
+- The same science product type  
+- The same processing level  
+- A consistent file structure   
+
+A collection is the *group* that defines **what** the dataset is.
+
+**Examples:**
+
+- *SMAP L4 Global 3-hourly 9 km EASE-Grid Surface and Root Zone Soil Moisture Geophysical Data V008*  
+- *ATLAS/ICESat-2 L3A Land and Vegetation Height V007*  
+
+Collection metadata includes at least the following:
+
+- Product description  
+- Spatial & temporal coverage  
+- Data provider
+- Variables  
+- Version 
+
+## 📄 Granules  
+
+A **granule** is a single file within a collection.
+
+Each granule usually represents a **single point in space and/or time**. For example, one granule could represent:
+
+- One SMAP orbit pass, or  
+- One ICESat-2 ground track   
```

Comment:
ICESat-2 may be a tricky example.  An IS2 ground track is a single beam that are named GT1L, GT1R, GT2L, GT2R, GT3L, GT3R.  For ATL03 and above, data for these 6 beams are in a single file, referenced by an imaginary reference ground track (RGT) that traces a line on the Earth surface between GT2L and GT2R.  RGTs are uniquely numbered 1 to 1387 for each 91 period.  Each 91 day period is referenced by a cycle number.  For archiving data from RGTs are split into 14 regions.  An ICESat-2 granule contains data for the 6 beams associated with an RGT for a given cycle for one of these regions.

https://www.earthdata.nasa.gov/learn/earth-observation-data-basics/glossary#ed-glossary-g